packages/playwright-core/types/types.d.ts

```diff
@@ -6512,46 +6512,7 @@ export interface BrowserContext {
    *
    * @param cookies
    */
-  addCookies(cookies: Array<{
-    name: string;
-
-    value: string;
-
-    /**
-     * either url or domain / path are required. Optional.
-     */
-    url?: string;
-
-    /**
-     * either url or domain / path are required Optional.
-     */
-    domain?: string;
-
-    /**
-     * either url or domain / path are required Optional.
-     */
-    path?: string;
-
-    /**
-     * Unix time in seconds. Optional.
-     */
-    expires?: number;
-
-    /**
-     * Optional.
-     */
-    httpOnly?: boolean;
-
-    /**
-     * Optional.
-     */
-    secure?: boolean;
-
-    /**
-     * Optional.
-     */
-    sameSite?: "Strict"|"Lax"|"None";
-  }>): Promise<void>;
+  addCookies(cookies: Array<NewCookie>): Promise<void>;
```

Comment:
This file is generated, so you should not edit it.

Reply:
I'll close this out for now then and wait to hear on #13237